recipes/itak/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "2.0.1" %}
+{% set name = "itak" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://lab.moilab.net/download/iTAK-{{ version }}.tar.gz
```

Comment:
Please consider using the URL from Github Releases for this, as it will allow the bot to automatically create PRs for future releases. You will also get an email notification when the PR is create

Reply:
URL from GitHub releases is good for me, but sometimes it can not be researched by users in mainland China, I will change it github first.